src/settings/settings.cpp

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// configuration.cpp
+//
+// Identification: src/configuration/configuration.cpp
+//
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+
+#include <gflags/gflags.h>
+#include "type/types.h"
+
+// gflags DEFINE
+#define __SETTING_GFLAGS_DEFINE__
+#include "settings/settings_macro.h"
+#include "settings/settings.h"
+#undef __SETTING_GFLAGS_DEFINE__
```

Comment:
I prefer to put all setting variables in settings.cpp. There are still some hard code variables in settings.cpp such as peloton_layout_mode and peloton_gc_mode. They are not controlled by gflags. Should we also move them into pg_settings?